config/config.go

```diff
@@ -122,7 +122,7 @@ func (l *LXD) GetInstanceType() LXDImageType {
 func (l *LXD) Validate() error {
 	if l.UnixSocket != "" {
 		if _, err := os.Stat(l.UnixSocket); err != nil {
-			return fmt.Errorf("could not access unix socket %s: %q", l.UnixSocket, err)
+			return fmt.Errorf("could not access unix socket %s: %w", l.UnixSocket, err)
```

Comment:
Note to self. Disable the Unix socket handling when running on Windows. Only remote TCP connections will work.